component/administrator/language/en-GB/com_jcomments.ini

```diff
@@ -255,7 +255,7 @@ AP_COMMENT_MINLENGTH="Minimum comment length"
 AP_COMMENT_MINLENGTH_DESC="The minimum number of characters a user should post. The user will get an alert if comment's length less then this value."
 AP_SHOW_COMMENTLENGTH="Character counter"
 AP_SHOW_COMMENTLENGTH_DESC="Enable/Disable displaying the character count while typing."
-AP_ENABLE_ACCEPT_TERMS_OF_USE="Don't show checkbox \"Accept terms of use\"."
+AP_ENABLE_ACCEPT_TERMS_OF_USE="Do not show the \"Accept terms of use\"."
```

Comment:
It is the other way around. Setting this option SHOWS the terms.

Therefore (beware typo in second): 
```
AP_ENABLE_ACCEPT_TERMS_OF_USE="Show \"Accept terms of use\" checkbox"
AP_ENABLE_ACCEPT_TERMS_OF_USE_DESC="Enable/Disable displaying the checkbox to accept terms of use."
```

Reply:
Can you fix it?

Reply:
https://github.com/exstreme/Jcomments-4/pull/61/commits/1ca19c839dc7fba886019f1a1d4f481d494cbb46 fixed